packages/frontend/admin/src/app.tsx

```diff
@@ -80,10 +80,10 @@ export const router = _createBrowserRouter(
               path: 'accounts',
               lazy: () => import('./modules/accounts'),
             },
-            // {
-            //   path: 'ai',
-            //   lazy: () => import('./modules/ai'),
-            // },
+            {
```

Comment:
not the time

Reply:
It only displays a line of hint on this page. 
"AI functionality is not currently supported. Self-hosted AI support is in progress."

Reply:
ok